src/elm/Update.elm

```diff
@@ -7,14 +7,15 @@ import Http
 import Maybe exposing (..)
```

Comment:
That line too you've also called `Json.Decode`, `D`, I'm wondering if that could get a little confusing with `Dict`, I'd stick to just aliasing it as `Json` (so it would be `Json.map`)